util/factions.ts

```diff
@@ -0,0 +1,14 @@
+export type Factions = "american" | "german" | "dak" | "british";
+
+export const getFactionName = (faction: Factions) => {
```

Comment:
Btw we have this over here:
https://github.com/cohstats/coh3-stats/blob/master/src/coh3/coh3-data.ts

and 
https://github.com/cohstats/coh3-stats/blob/master/src/coh3/coh3-types.ts

We can change this later :)